docs/dictionary/function/shell.lcdoc

```diff
@@ -16,22 +16,22 @@ OS: mac, windows, linux
 Platforms: desktop, server
 
 Example:
-shell("ls -l *.txt") -- returns a listing of the current dir on Unix
+put shell("ls -l *.txt") into myList -- returns a listing of the current dir on Unix
```

Comment:
I think our variable convention should probably be kept as (eg) `tList` rather than `myList` as much as possible in the docs... Either way the variable needs to be declared in the example.